Classes/Service/Lucene.php

```diff
@@ -199,20 +199,37 @@ public function find($searchTerm, &$query = null) {
 			
 			// Extend / rewrite the search query
 			$tokenTerms			= array();
-			$searchTerm			= $this->_rewriteQueryTerms($searchTerm, $tokenTerms);
-			
+			$luceneSearchTerm	= $this->_rewriteQueryTerms($searchTerm, $tokenTerms);
+
 			try {
 				$highlight		= intval(\Tollwerk\TwLucenesearch\Utility\Indexer::indexConfig($GLOBALS['TSFE'], 'search.highlightMatches'));
 				
 				// Construct the search request
-				$query			= $this->query($searchTerm);
+				$query			= $this->query($luceneSearchTerm);
 				
 				// Run the search an collect the results
 				$hits			= new \Tollwerk\TwLucenesearch\Domain\Model\QueryHits($this->_index(), $query, $tokenTerms, (boolean)$highlight);
 				
 			// In case of errors: Invalidate the query hits
 			} catch(\Zend_Search_Lucene_Exception $e) {
 				$hits			= null;
+

```

Comment:
@jkphl Eigentlich gehört dieses Commit nicht wirklich in den PR und ist wohl nachträglich ausversehen reingerutscht, als ich diese Ergänzung für ein Projekt vorgenommen habe.

Aber natürlich gab es die Ergänzung nicht ohne Grund und vielleicht kann man sie dennoch übernehmen oder eine alternative Lösung für das damit gelöste Problem finden.

Das Problem war, dass eine Suche mit gewissen Zeichen zu keinem Ergebnis führte.
Z.b. eine Suche nach einem Wort mit Bindestrich "-" oder einer E-Mail Adresse.